src/payment/strategies/moneris/moneris-payment-strategy.ts

```diff
@@ -128,18 +128,21 @@ export default class MonerisPaymentStrategy implements PaymentStrategy {
             display_labels: 1,
             enable_exp: 1,
             enable_cvd: 1,
-            css_body: style?.cssBody || 'background:transparent;',
-            css_textbox: style?.cssTextbox || 'border-width:2px;',
-            css_textbox_pan: style?.cssTextboxCardNumber || 'width:140px;',
-            css_textbox_exp: style?.cssTextboxExpiryDate || 'width:40px;',
-            css_textbox_cvd: style?.cssTextboxCVV || 'width:40px',
+            css_body: style?.cssBody || 'font-family: Arial, Helvetica,sans-serif;background:transparent;',
+            css_textbox: style?.cssTextbox || 'border-radius:4px;border:2px solid rgb(00,00,00);width:100%;font-weight:600;padding:8px 8px;outline:0;',
```

Comment:
Just one question is there a reason why we don't have spacing between between css property and values?

Reply:
@animesh1987  I removed because the CSS is sent to the iframe in the http request and i don't want to add more characters but i tested and is working with the extra spaces

Reply:
♻️  @animesh1987